Formula/curl-openssl.rb

```diff
@@ -0,0 +1,52 @@
+class CurlOpenssl < Formula
+  desc "Get a file from an HTTP, HTTPS or FTP server"
+  homepage "https://curl.haxx.se/"
+  url "https://curl.haxx.se/download/curl-7.62.0.tar.bz2"
+  mirror "http://curl.mirror.anstey.ca/curl-7.62.0.tar.bz2"
+  sha256 "7802c54076500be500b171fde786258579d60547a3a35b8c5a23d8c88e8f9620"
+
+  keg_only :versioned_formula
```

Comment:
I'd rather have a message that hints this is a different version of `curl` vs a formula that isn't linked because it is also provided by macOS. Same sort of logic for `keg_only` reason for `nghtt2-libonly`.

I guess this would be the start of a discussion about how the maintainers would like to handle these kinds of formulae now that formula options are discouraged.

Reply:
I don't think we will be seeing a lot of these types of formulas so this is going to be an exception. Technically its not a different version in the same sense `versioned_formula` is normally used though since this should be at the same level as the regular curl formula. I guess it doesn't really make a big difference since the output is the same with the exception of one string.